### DISCUSSION TITLE
Solution precision in sub-monthly GRACE/GRACE-FO gravity field recovery

### DISCUSSION THREAD

**Discussion by huiliyikk** [groops-devs/groops#123] [Q&A] [answered]
Dear developers,

I used GROOPS for monthly solution and it went well. And I want to know if it is possible for GROOPS to solve 10days' or less than 

10days' solution, cuz the 10days' solution with 40degrees didn't seem ordinary.

regards
lyh
![2021-01-01degreeVariances2](https://user-images.githubusercontent.com/62589660/198873270-59e8ff33-8724-4d59-a472-522bc9e0ef24.png)

**Comment by akvas** [comment, marked as answer]
Hi @huiliyikk,

conceptionally there is no difference between a 10-day and a monthly solution and GROOPS is able to compute both (or even shorter time spans). Just from the reduction in data volume you would expect a $\sqrt{3}$ decrease in accuracy which seems to be the case in your plot. 

Cheers,
Andreas
